public/locations-list-map.js

```diff
@@ -596,7 +596,7 @@ function loadMapScript(searchParams, data, filters) {
   const scriptTag = ce('script');
 
   // API Key below is only enabled for *.findthemasks.com/* Message @susanashlock for more info.
-  const apiKey = 'AIzaSyDSz0lnzPJIFeWM7SpSARHmV-snwrAXd2s';
+  const apiKey = window.GOOGLE_MAPS_API_KEY || 'AIzaSyDSz0lnzPJIFeWM7SpSARHmV-snwrAXd2s';
```

Comment:
this is for overriding in dev/staging?

Reply:
This is for local env so it can still work without running `local_install.sh` (i.e. for people developing on Windows).